package.json

```diff
@@ -7,15 +7,17 @@
   "scripts": {
     "dev": "webpack-dev-server --inline --progress --config build/webpack.dev.conf.js",
     "start": "npm run dev",
-    "lint": "eslint --ext .js,.vue src",
+    "lint": "eslint --fix --ext .js,.vue src",
     "build": "node build/build.js",
     "deploy": "npm run build; push-dir --dir=dist --branch=gh-pages --cleanup"
   },
   "dependencies": {
+    "axios": "^0.18.0",
     "vue": "^2.5.2",
     "vue-router": "^3.0.1",
     "vue-social-sharing": "^2.3.3",
-    "vuetify": "^1.1.13"
+    "vuetify": "^1.1.13",
+    "vue-carousel": "^0.10.0"
```

Comment:
Vuetify also has a carousel component and we are already using vuetify, so can you use vue-carousel to reduce an extra dependency?

Reply:
Vue-carousal has features that vuetify-carousal don't provide.For ex:Vuetify carousal don't allow to load multiple slides on one screen and then resposively change it to one slide per screen on smaller screen.